### ISSUE TITLE
Very promising  

### ISSUE THREAD

**Issue by rivman** [dansleboby/LMPA#1]
Hello. 
Very promising, thank you for sharing. How about making it i18n ready ? I could help with 2 languages already (French, Romanian). You could use simple language arrays, or even gettext.
Have a great day !

[Label: ENHANCEMENT]

**Comment by dansleboby** [comment]
Yes it will be great, but I will also need to add some sort of setting to keep to choosen local, also I'm still on very early stage and I need to make more test to get a beta release :)
